src/MainnetController.sol

```diff
@@ -274,6 +286,72 @@ contract MainnetController is AccessControl {
         );
     }
 
+    /**********************************************************************************************/
+    /*** wstETH Integration                                                                     ***/
+    /**********************************************************************************************/
+
+    function depositToWstETH(uint256 amount) external {
+        _checkRole(RELAYER);
+        _rateLimitedAsset(LIMIT_WSTETH_DEPOSIT, Ethereum.WETH, amount);
+
+        proxy.doCall(
+            Ethereum.WETH,
+            abi.encodeCall((IWETH(Ethereum.WETH)).withdraw, (amount))
+        );
+
+        proxy.doCallWithValue(
+            Ethereum.WSTETH,
+            "",
+            amount
+        );
+    }
+
+    function requestWithdrawFromWstETH(uint256 amount) external returns (uint256[] memory) {
+        _checkRole(RELAYER);
+        _rateLimitedAsset(LIMIT_WSTETH_REQUEST_WITHDRAW, Ethereum.WSTETH, amount);
+
+        proxy.doCall(
+            Ethereum.WSTETH,
+            abi.encodeCall(IERC20(Ethereum.WSTETH).approve, (Ethereum.unstETH, amount))
```

Comment:
unstETH should be called WSTETH_WITHDRAW_QUEUE and be moved to its own section in spark-address-registry like:

```solidity
    /******************************************************************************************************************/
    /*** Superstate Addresses                                                                                       ***/
    /******************************************************************************************************************/

    address internal constant SUPERSTATE_REDEMPTION = 0x4c21B7577C8FE8b0B0669165ee7C8f67fa1454Cf;

```